src/main/java/com/sarahhand/fractals/juliaset/JuliaConfig.java

```diff
@@ -136,11 +150,18 @@ public JuliaConfig(Double center, double zoom, int maxDwell, ColorPalette palett
 	public JuliaConfig(@JsonProperty("x") double x, @JsonProperty("y") double y, 
 			@JsonProperty("zoom") double zoom, 
 			@JsonProperty("maxDwell") int maxDwell, 
-			@JsonProperty("palette") ColorPalette palette) {
+			@JsonProperty("palette") ColorPalette palette,
```

Comment:
Update the Javadoc to include the new parameter and a description of what they are.